### ISSUE TITLE
Fire isn't synced correctly

### ISSUE THREAD

**Issue by thegrb93** [Regalis11/Barotrauma#721]
People don't know they're too close to the fire and dying until they're dead.

**Comment by rico6822** [comment]
The very smoke of the fire is already killing very fast.

**Comment by BlueTheKing** [comment]
Heat from the fire?

**Comment by NilanthAnimosus** [comment]
For the sake of adding to this having noticed it:

NilMod hosts can change the rates of fire growth and have completely different rates of growth, shrinking and such.

NilMod hosts can also modify the damage value of fire, along with giving it a modified damage range, I changed quite a bit of that code to make it possible.

It isn't buggy in my mod strictly, it syncs rather well but the host must enable syncing fires on the mod (Vanilla does not sync fires I believe, and my code the hull and thus fires can be synced periodically based on a timer and amount changed), I did not see a reason to sync fires for all hosts using the mod when most use vanilla mechanics and vanilla values, since that'd be a waste of bandwidth and events.

as far as I know, the fire syncs perfectly in vanilla, and so does room oxygen %. 

Fire does in vanilla have a considerable burn range to it though, I am not entirely sure if this is really a networking issue as the title suggests since the description is a little lacking to be more specific - if you couldn't see the fire that'd be a large issue, but generally if your closer than the fire extinguisher's rough range your probably within range to take some considerable damage.
